x-pack/plugins/observability/public/components/shared/exploratory_view/components/action_menu/action_menu.test.tsx

```diff
@@ -12,14 +12,23 @@ import { sampleAttribute } from '../../configurations/test_data/sample_attribute
 import * as pluginHook from '../../../../../hooks/use_plugin_context';
 import { TypedLensByValueInput } from '@kbn/lens-plugin/public';
 import { ExpViewActionMenuContent } from './action_menu';
+import { noCasesPermissions as mockUseGetCasesPermissions } from '../../../../../utils/cases_permissions';
 
 jest.spyOn(pluginHook, 'usePluginContext').mockReturnValue({
   appMountParameters: {
     setHeaderActionMenu: jest.fn(),
   },
 } as any);
 
+jest.mock('../../../../../hooks/use_get_user_cases_permissions', () => ({
```

Comment:
I was getting errors relating to `useGetUserCasesPermissions` because it is now dependent on the cases plugin so I'm mocking the whole thing now.